src/daemons/linuxd/src/control_plane.rs

```diff
@@ -40,6 +40,7 @@ pub fn try_read_command(stream: &mut SocketStream) -> Result<Command, SocketErro
     })
```

Comment:
As discussed offline, we should maybe consider this moving to a library eventually as I don't think this module belongs in `linuxd`.

But we can leave this to a different PR.